aten/src/ATen/Context.cpp

```diff
@@ -332,8 +332,8 @@ const std::vector<at::QEngine>& Context::supportedQEngines() {
 
 #ifdef USE_FBGEMM
     if (fbgemm::fbgemmSupportedCPU()) {
-      // The X86 qengine is available if and only if FBGEMM is available
       engines.push_back(at::kX86);
```

Comment:
actually this PR is not making x86 the default quantized engine yet, I believe with the current code you'll need to switch the order, but I have https://github.com/pytorch/pytorch/pull/89804, and maybe you can change the default more explicitly after my PR is landed

Reply:
Since this PR was already landed, I changed the PR title to avoid misunderstanding. @XiaobingSuper Could you please follow up the changes after #89804 is landed? Thanks!